src/main/java/com/conveyal/gtfs/GTFSFeed.java

```diff
@@ -233,9 +236,9 @@ public void toFile (String file) {
             new StopTime.Writer(this).writeTable(zip);
 
             if (!this.bookingRules.isEmpty()) new BookingRule.Writer(this).writeTable(zip);
-            if (!this.locationGroups.isEmpty()) {
-                // export location groups
-                JdbcGtfsExporter.writeLocationGroupsToFile(zip, new ArrayList<>(locationGroups.values()));
+            if (!this.stopAreas.isEmpty()) {
```

Comment:
It's because the table can have more than one entry for an area id. If you take the example here: https://github.com/google/transit/issues/338, `stopAreas.getCsvReader` will group by area id to produce: 

`group1, "stop 1, stop 2, zone A"`
`group2, "stop3, stop4"`

`stopAreas.packStopAreas` reverses this, back to mach the example. This is to aid CRUD by the UI.